src/common/Common.UI/OSVersionHelper.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Common.UI
+{
+    public static class OSVersionHelper
+    {
+        public static bool IsWindows11()
```

Comment:
Thought we already have & use one for the Settings?

Reply:
Yes, this is removing it from Settings and adding to Common.UI so it can be used on more apps.